src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/TextFilter.php

```diff
@@ -82,16 +82,11 @@ public function addAttributeFilter(
                 $mustNotClause = [
                     'query_string' => [
                         'default_field' => $attributePath,
-                        'query'         => '*' . $value . '*',
+                        'query'         => '*' . $escapedValue . '*',
                     ],
                 ];
 
-                $filterClause = [
```

Comment:
I'll check with Delphine and make another PR in the meantime for this particular operator